cmd/influxdb-writer/main.go

```diff
@@ -51,7 +54,19 @@ func main() {
 		log.Fatalf("failed to init logger: %s", err)
 	}
 
-	pubSub, err := brokers.NewPubSub(cfg.BrokerURL, "", logger)
+	tracer, traceCloser, err := jaegerClient.NewTracer(svcName, cfg.JaegerURL)
+	if err != nil {
+		logger.Fatal(fmt.Sprintf("failed to init Jaeger: %s", err))
+	}
+	defer traceCloser.Close()
+
+	pbTracer, traceCloser, err := jaegerClient.NewTracer(svcName, cfg.JaegerURL)
```

Comment:
It is unclear what is the difference between `tracer` and `pbTracer`?

If you are redefining `traceCloser`, how the first one will be closed with defer, at the end of the function? If I am not wrong, only second one will be closed, handle to the first one will be lost...

@dborovcanin what do you think?

Reply:
The idea for the two tracers was to distinguish between traces in the service and pub/ sub. The service name was initially different but from one of the comments I changed it to the svcName constant. But right now the two tracers have the same configuration, so it is best to remove the additional one.